package.json

```diff
@@ -203,7 +210,9 @@
     ],
     "snapshotSerializers": [
       "./test/newline-snapshot-serializer.js"
-    ]
+    ],
+    "testPathIgnorePatterns": ["/node_modules/", "/.cache/", "/dist/"],
```

Comment:
Is this necessary as part of this PR, or an unrelated optimization?

Reply:
It's an optimization, i can make a seperat pr for it, i had set the baseDir to that, so jest has found some tests in the checked out repositories.

Reply:
removed